application/plugin_base/terminal.py

```diff
@@ -1,9 +1,11 @@
 import gi
 
-# require specific version of Vte
-gi.require_version('Vte', '2.90')
+from gi.repository import Gtk, Gdk, Vte
 
-from gi.repository import Gtk, Gdk, Vte, GConf
+try:
+	from gi.repository import GConf as gconf
```

Comment:
I see no reason to import as a different name. In fact, every other import has been regular from `gi.repository`. The fact we are making it a conditional import doesn't not seem to warrant making the name lowercase.

As for lowercase, it's a PEP8 recommendation. As comments are not treated as sentences, they therefore have no need to start with upper case.